### ISSUE TITLE
IR information

### ISSUE THREAD

**Issue by oblitum** [tommitytom/wiimote#1]
Hi, nice modern lib for Wiimote you started :-)

I've looked around and couldn't find reading IR position, is it supported?

**Comment by tommitytom** [comment]
Hi!  Thanks for checking it out!  It's probably not very well documented and I can't remember if there were any strange bugs lurking so best of luck!

I did not implement IR as I don't use it very much, it shouldn't be too tough to add though.  I'm pretty busy right now but may be able to take a look sometime during the week and see whats involved.

Also a quick note - I haven't tested this with standard bluetooth, only the Dolphin Bar!

**Comment by oblitum** [comment]
OK, thanks anyway for the info. I'm checking others for Windows at the moment, it seems the sole one C++ that supports this is the old WiiYourself...

**Comment by tommitytom** [comment]
Yeah I remember using that back in the day!  My main reason for creating this one was for the Dolphin Bar support - the best I could find was code scattered amongst Dolphin that I tried to unify in one place

**Comment by tommitytom** [comment]
OK it shouldn't be too tough to implement, leaving these here as reference material:
https://github.com/Flafla2/Unity-Wiimote/blob/master/Assets/Wiimote/Scripts/Wiimote.cs#L275
https://github.com/Flafla2/Unity-Wiimote/blob/master/Assets/Wiimote/Scripts/Wiimote.cs#L481
https://github.com/Flafla2/Unity-Wiimote/blob/master/Assets/Wiimote/Scripts/Wiimote.cs#L732
https://github.com/Flafla2/Unity-Wiimote/blob/master/Assets/Wiimote/Scripts/WiimoteData/IRData.cs
